### PR TITLE
[4.1] Fix option order in select

### PR DESCRIPTION
**Pull request by pxpm**
REFS: #2847

### CODE REVIEW

**Comment by tabacitu** [comment]
Thanks for the talk. Note - how about if we populate the `<options>` with PHP, instead of JS? Like we do in `select2` and `select2_multiple`? Would that be an alternative solution?

If not, this is a clean solution too, we'll merge as-is.

**Comment by pxpm** [comment]
ping @tabacitu 
Removed js options, we get them directly from php. No more order worries.

**Comment by pxpm** [comment]
Test inside repeatable.

File: src/resources/views/crud/fields/relationship/relationship_select.blade.php [hunk 1]
```diff
@@ -80,6 +77,16 @@
...
+            <option value="">-</option>
+        @endif
+
+        @if (count($field['options']))
+            @foreach ($field['options'] as $key => $option)
+                    <option value="{{ $key }}">{{ $option }}</option>
+            @endforeach
+        @endif
+    </select>
     </select>
```
Comment (tabacitu) [inline src/resources/views/crud/fields/relationship/relationship_select.blade.php:90]: I see there are two endings to the `<select>` block. The HTML block.

File: src/resources/views/crud/fields/relationship/relationship_select.blade.php [hunk 2]
```diff
@@ -80,6 +77,16 @@
...
+        @if ($field['allows_null'])
+            <option value="">-</option>
+        @endif
+
+        @if (count($field['options']))
+            @foreach ($field['options'] as $key => $option)
+                    <option value="{{ $key }}">{{ $option }}</option>
+            @endforeach
+        @endif
+    </select>
```
Comment (pxpm) [inline src/resources/views/crud/fields/relationship/relationship_select.blade.php:89]: remove

**Comment by pxpm** [comment]
@tabacitu merge please. Works good inside repeatable and inline.

**Pull request merged**